dlio_benchmark/checkpointing/base_checkpointing.py

```diff
@@ -86,22 +240,55 @@ def get_layer_index(self, rank, tensor_parallelism, pipeline_parallelism, total_
             start_layer = left_layers * max_layer_per_pipeline + (pipeline_rank - left_layers) * (min_layers_per_pipeline)
             end_layer = start_layer + num_layers_per_pipeline - 1
         return start_layer, end_layer
+        '''
```

Comment:
I am fine with your removing the comment as we have git to check what the code was.